client/components/create-account.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+export default class CreateAccount extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      imgSrc: null,
+      username: ''
+    };
+    this.handleUpload = this.handleUpload.bind(this);
+    this.handleUsernameChange = this.handleUsernameChange.bind(this);
+  }
+
+  handleUpload(event) {
+    this.setState({ imgSrc: URL.createObjectURL(event.target.files[0]) });
+  }
+
+  handleUsernameChange(event) {
+    this.setState({ username: event.target.value });
+  }
```

Comment:
This component is finished when it actually register's the user in the database.